plugins/out_stackdriver/stackdriver.c

```diff
@@ -461,6 +469,11 @@ static int pack_json_payload(bool operation_extracted, int operation_extra_size,
         msgpack_object_kv* kv = obj->via.map.ptr;
         msgpack_object_kv* const kvend = obj->via.map.ptr + obj->via.map.size;
         for(; kv != kvend; ++kv	) {
+            if (insertId_extracted && strncmp(INSERTID_IN_JSON,kv->key.via.str.ptr, kv->key.via.str.size) == 0 
```

Comment:
```suggestion
            if (insertId_extracted && strncmp(INSERTID_IN_JSON, kv->key.via.str.ptr, kv->key.via.str.size) == 0 
```

Reply:
I hope we just have a lint/formatter so we don't need to worry about it :)

Reply:
Done.